### ISSUE TITLE
PTY resize test flake

### ISSUE THREAD

**Issue by petervo** [cockpit-project/cockpit#7566]
Seems to happen more frequently though not always on non 86_64 arches. 

https://kojipkgs.fedoraproject.org/work/tasks/5995/21475995/build.log

```
cockpit-bridge:ERROR:src/bridge/test-pipe-channel.c:689:test_spawn_pty_resize: '*{4567 1234}*{42 24}*' does not match: echo -e "\x7b$COLUMNS $LINES\x7d"
]0;<mock-chroot>[mockbuild@buildvm-ppc64-11 cockpit-149]$ echo -e "\x7b$COLUMNS $LINES\x7d"
{4567 1234}
]0;<mock-chroot>[mockbuild@buildvm-ppc64-11 cockpit-149]$ echo -e "\x7b$COLUMNS $LINES\x7d"
{4567 1234}
]0;<mock-chroot>[mockbuild@buildvm-ppc64-11 cockpit-149]$ exit
```

**Comment by larskarlitski** [comment]
I can't reproduce this here. I've enabled `checkwinsize` in #7582 to see if that might fix it.

The problem seems to be that `$COLUMNS` and `$LINES` don't get updated with the ioctl call. It could also be that there's a race between `SIGWINCH` and the echo call in the test. Not sure how we could work around that other than `sleep()`ing.

**Comment by KKoukiou** [comment]
This is a flake reported 2 years ago and I never saw it for the whole year I am part of the Cockpit team. If someone sees it again let's reopen.

**Comment by martinpitt** [comment]
Haha, I closed the duplicate #11454 a few hours ago as well. :-)
